tooling/helpers/file_helper.py

```diff
@@ -12,14 +12,31 @@ def load_file_list(size: str):
         if os.path.isfile(os.path.join(dirpath, f)) and f.endswith(".wav")
     ]
 
+def load_export_file_list():
+    return [
+        os.path.join(EXPORT_BASE_PATH, f)
+        for f in os.listdir(EXPORT_BASE_PATH)
+        if os.path.isfile(os.path.join(EXPORT_BASE_PATH, f)) and f.endswith(".json")
+    ]
+
 
-def get_expected_transcription(path: str):
-    json_path = f"{path[:-4]}.json"
+def get_corresponding_transcript(export_path: str, scale: str):
+    path = export_path.replace(f"{EXPORT_BASE_PATH}", f"{os.path.join(DATA_BASE_PATH, scale)}")
 
-    with open(json_path) as f:
+    with open(path) as f:
         d = json.load(f)
         return str(d["transcript"])
 
-
 def get_file_name(path, scale):
     return path.replace(f"{os.path.join(DATA_BASE_PATH, scale)}/", "")
+
+def get_file_id(export_dir_path):
+    p = export_dir_path.replace(f"{EXPORT_BASE_PATH}/", "")
+    return p[:len(p)-5]
```

Comment:
-5 is somewhat a magic number
I would recommend explicitly removing .json (I am assuming that is happening - see it's unclear)

```suggestion
    return p.removesuffix(".json")
```